_sass/components/_side-nav.scss

```diff
@@ -40,17 +40,17 @@
 @include media($nav-width) {
   .usa-layout-docs {
     display: block;
-  }
 
-  .sidenav {
-    display: block;
-    float: left;
-    padding-right: 1rem;
-  }
+    .sidenav {
+      display: block;
+      float: left;
+      padding-right: 1rem;
+    }
+    .usa-layout-docs-main_content:nth-child(2) {
```

Comment:
only applies this style when `.usa-layout-docs-main_content` is the 2nd child of `.usa-layout-docs`